src/ControlSurface/Controls/Position/ScorePosition.qml

```diff
@@ -109,30 +116,24 @@ Rectangle {
     Connections {
         // Adding a colorpicker in the control surface
         function onAppendPosition(s) {
-            function find(cond) {
-                for (var i = 0; i < positionListModel.count; ++i)
-                    if (cond(positionListModel.get(i)))
-                        return i
-                return null
-            }
-            var a = find(function (item) {
+            //the index of m.Path in the listmodel
+            var a = Utility.find(positionList.model, function (item) {
                 return item.id === JSON.stringify(s.id)
-            }) //the index of m.Path in the listmodel
+            })
             if (a === null) {
                 var tmpX = s.Value.Vec2f[0]
                 var tmpY = s.Value.Vec2f[1]
                 var tmpMin = s.Domain.Float.Min
                 var tmpMax = s.Domain.Float.Max
                 var tmpDomain = Math.abs(tmpMax - tmpMin)
-                // vertical.x = tmpX * ( positionBackground / tmpDomain )
-                // horizontal.y = tmpY * ( positionBackground / tmpDomain )
                 positionListModel.insert(positionListModel.count, {
                                              "_custom": s.Custom,
                                              "_id": s.id,
                                              "_uuid": s.uuid,
                                              "_controlX": tmpX,
                                              "_controlY": tmpY,
-                                             "_controlDomain": tmpDomain
+                                             "_controlDomain": tmpDomain,
+                                             "_controlMin": tmpMin
```

Comment:
why no max in there ?

Reply:
because max is included in controlDomain = max-min

Reply:
oh, then we'd say range instead of domain. But not dramatic